api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -1081,6 +1081,8 @@ public static String getOpenmrsProperty(String property) {
 		
 		props
 				.add(new GlobalProperty(GP_DRUG_ORDER_DRUG_OTHER, "", "Specifies the uuid of the concept which represents drug other non coded"));
+		props.add(new GlobalProperty(LOGIN_URL, "login.htm",
```

Comment:
Do you need to write login.htm in two places? 😊

Reply:
Nope, one of them is just a description of the key pair value of the global properties.